verify/basic_test.go

```diff
@@ -88,5 +91,20 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(sshSession).To(Exit(3))
 		Expect(sshSession.outputToString()).To(Equal("inactive"))
+
+		// set by podman-rpm-info-vars.sh
+		if version := os.Getenv("PODMAN_VERSION"); version != "" {
```

Comment:
@baude I wonder if we should error if not set. But then I guess testing locally might be more annoying as you always would need to set this to the right version.

Because right now this might be NOP if CI no longer set this var for whatever reason.

Reply:
right, agree